client/common/directives/ModelFieldPointsOfInterest/ModelFieldPointsOfInterest.js

```diff
@@ -100,6 +100,8 @@ angular.module('dashboard.directives.ModelFieldPointsOfInterest', [
 				var markerLocation;
 				var infowindow;
 				var requestQuery;
+				var isfinished = false;
```

Comment:
this should be camel case though it looks like it's not being used anywhere this should be removed